src/app/modules/common/services.js

```diff
@@ -342,7 +361,7 @@ angular.module('BloomLibraryApp.services', ['restangular'])
 		// The caller will typically do getFilteredBooksCount(...).then(function(count) {...}
 		// and inside the scope of the function count will be the book count.
 		// See comments in getFilteredBookRange for how the parse.com query is mapped to an angularjs promise.
-		this.getFilteredBooksCount = function (searchString, shelf, lang, tag) {
+		this.getFilteredBooksCount = function (searchString, shelf, lang, tag, includeOutOfCirculation) {
             $analytics.eventTrack('Book Search', {searchString: searchString || '', shelf: (shelf && shelf.name) || '', lang: lang || '', tag: tag || ''});
```

Comment:
I don't see includeOutOfCirculation defined or initialized anywhere in this scope